pkg/util/util.go

```diff
@@ -0,0 +1,40 @@
+package util
+
+import (
+	"slices"
+	"unsafe"
+)
+
+// Allocation free conversion from []byte to string (unsafe)
+// Note that the byte slice must not be modified after conversion
+func ByteSliceToString(bs []byte) string {
+	return unsafe.String(unsafe.SliceData(bs), len(bs))
```

Comment:
I did a bit of a double-take at seeing this neat trick-- apparently we've had [`unsafe.StringData`](https://pkg.go.dev/unsafe#StringData) and [`unsafe.Slice`](https://pkg.go.dev/unsafe#Slice) since Go 1.20! Very cool. :sunglasses:

Reply:
I know, right? I don't think I've ever modified a byte slice after calling `string(..)` on it... so this feels pretty much like a no-brainer at least for my own projects.

Btw, fun to see you show up here :) Was it @srenatus who snitched about my whereabouts? 😄